### ISSUE TITLE
Remove "bookmarks" from Project model 

### ISSUE THREAD

**Issue by maxceem** [topcoder-platform/tc-project-service#471]
We are moving links to be stored in the `ProjectAttachmenets` model instead of the `bookmarks` field in the `Project` model, see respective ticket https://github.com/appirio-tech/connect-app/issues/3474.

As soon as we successfully transfer all the links in existent data to the new model and test it out for some time on DEV and PROD, we have to remove all the old logic regarding `bookmarks`.

**Comment by maxceem** [comment]
@vikasrohit do you think we can do this now? A new way of storing links in the attachments looks like working well. Or would you like to hold on it a little bit more?

**Comment by vikasrohit** [comment]
Yep, we can do it now. But lets schedule it for on release after.
